WSLBlobNFS/WSLBlobNFS.psm1

```diff
@@ -1,205 +1,676 @@
-# Set the execution policy to appropriate value to run the script
-# Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope CurrentUser
+# --------------------------------------------------------------------------------------------
```

Comment:
one of the high level todos is to allow the smb volume (plus nfs volume) to be automounted when user restarts wsl for some reason. Equivalent of /etc/fstab in Linux